packages/infero/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+#
+from spack import *
+
+
+class Infero(CMakePackage):
+    '''
+    Infero runs a pre-trained ML model for inference. It can be deployed on a HPC system 
+    without the need for high-level python libraries (e.g. TensorFlow, PyTorch, etc..)
+    '''
+
+    ecmwf = "https://github.com/ecmwf-projects/infero.git"
+
+    version('0.1.2',
+            git=ecmwf,
+            commit='4c229a16ce75a249c83cbf43e0c953a7a42f2f83')
+
+    maintainers = ['juckerj']
+
+    variant('quiet', description='Disable calls to Log::info', default=False)
+
+    depends_on('eckit@1.20.0:')
+    depends_on('fckit')
+    depends_on('ecbuild', type=('build'))
+    depends_on('tensorflowc')
+
+    patch('comment_out_log-level_info.patch', when='+quiet')
```

Comment:
Isn't that patch only valid for `@0.1.2`? Because I imagine that code base changed over time and keeps changing, so that patch can't be valid for all versions, right?

Reply:
you are right!